frontend/socialapp/my-pov-app/app/services/auth/AuthService.tsx

```diff
@@ -0,0 +1,102 @@
+import { saveData, deleteData } from "app/utils/storage/securestore"
+
+const API_URL = "http://10.0.2.2:8000/users" // Backend API URL
+const login = async (userData: {}) => {
+  try {
+    const response = await fetch(`${API_URL}/login/`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify(userData),
+    })
+    const jsonData = await response.json()
+    if (!response.ok) {
+      throw new Error(jsonData.error)
+    }
+    saveData("accessToken", jsonData.access).catch((error) => {
+      throw new Error(error)
+    })
+    saveData("refreshToken", jsonData.refresh).catch((error) => {
+      throw new Error(error)
+    })
+    return Promise.resolve()
+  } catch (error: any) {
+    console.log("Error fetching data:", error.message)
```

Comment:
i dont know if this is the best way to handle errors 
seems a bit weird to check if the response is not resolved and then returning the jsonData of the unresolved response (response with error) 
Does this log the http error code?
This is important because we need to know where the response faced an error